Rlabkey/DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: Rlabkey
-Version: 2.5.2
+Version: 2.5.3
 Date: 2020-08-05
```

Comment:
Good catch, I'll get that updated. I did some local testing but it would be great if you could pull down the changes and perform some manual testing on your own.

Reply:
Sure thing. Testing locally now.